deployments/ecmps/release-v2.0-fix/Sprint8/6111/camdecmps/functions/rpt_quartely_em_summary_data.sql

```diff
@@ -0,0 +1,331 @@
+                    --
+                    -- Quarterly Data for the Non-Fuel-Specific and Fuel-Specific Report Report Tables
+                    --
+                    select  sel.mon_plan_id,
+                            sel.fuel_ind,
+                            sel.rpt_period_id,
+                            sel.mon_loc_id,
+                            prd.calendar_year,
+                            ( 'QTR ' || prd.quarter ) as Period,
+                            case 
+                                when sel.fuel_ind = 0
+                                then null
+                                else coalesce( osd.fuel_cd, 'NO DATA' )
+                            end as fuel_cd,
+                            max( case when osd.op_type_cd = 'OPHOURS'   then osd.Op_Value end ) as Op_Hours,
+                            max( case when osd.op_type_cd = 'OPDAYS'    then osd.Op_Value end ) as Op_Days,
+                            max( case when osd.op_type_cd = 'OPTIME'    then osd.Op_Value end ) as Op_Time,
+                            max( case when osd.op_type_cd = 'HIT'       then osd.Op_Value end ) as Hit,
+                            max( case when osd.op_type_cd = 'SO2M'      then osd.Op_Value end ) as So2m,
+                            max( case when osd.op_type_cd = 'CO2M'      then osd.Op_Value end ) as Co2m,
+                            max( case when osd.op_type_cd = 'NOXR'      then osd.Op_Value end ) as Noxr,
+                            max( case when osd.op_type_cd = 'NOXM'      then osd.Op_Value end ) as Noxm,
+                            max( case when osd.op_type_cd = 'BCO2'      then osd.Op_Value end ) as Bco2
+                    from  (
+                                select  lst.mon_plan_id,
+                                        lst.rpt_period_id,
+                                        lst.mon_loc_id,
+                                        ind.fuel_ind,
+                                        -- Indicates whether fuel specific data was reported for the location in the emission report
+                                        max(case when osd.fuel_cd is null then 0 else 1 end) as Fuel_Exists_Ind
+                                from  lst
+                                        join (select 0 as fuel_ind) ind
+                                        on null is null
+                                        join camdecmps.operating_supp_data osd
+                                        on osd.mon_loc_id = lst.mon_loc_id
+                                        and osd.rpt_period_id = lst.rpt_period_id
+                                        and osd.op_type_cd in ( 'BCO2', 'CO2M', 'HIT', 'NOXM', 'NOXR', 'OPDAYS', 'OPHOURS', 'OPTIME', 'SO2M' )
+                                group
+                                    by  lst.mon_plan_id,
+                                        lst.rpt_period_id,
+                                        lst.mon_loc_id,
+                                        ind.fuel_ind
```

Comment:
FUEL_IND does not appear to be needed since it should always be 0 for the non-fuel-specific information.